src/views/Bitcoin/Xpub.js

```diff
@@ -55,9 +41,25 @@ export default function Xpub() {
         <Grid.Item full className="mt4" as={Text}>
           Paste it into landscape while setting up your wallet.
         </Grid.Item>
+        <Grid.Item
+          full
+          className="mt4"
+          as={Text}
+          style={{ fontSize: '14px', color: 'gray' }}>
+          {btcMainnet ? 'BTC Mainnet' : 'BTC Testnet'}
+        </Grid.Item>
+        <Grid.Item full className="mt1">
+          <ToggleSwitch
+            selected={btcMainnet}
+            onClick={() => {
+              btcMainnet ? setXpub(vpub) : setXpub(zpub);
+              setBtcMainnet(!btcMainnet);
+            }}
+          />
+        </Grid.Item>
```

Comment:
We don't currently show network selection/toggles like this anywhere in the UI, and I don't think we want to present it to the user in most cases. Perhaps we should only render this if we're running in a development environment?

Reply:
Yeah you're right. I scrapped the toggle entirely and left a boolean that can be toggled manually when developing. This also allows me to remove all the dependency changes. The SignTransaction screen will still be able to sign BTC testnet transactions tho, but that should be fine since it parses whether we are on mainnet or testnet from the unsigned transaction itself.